docs/xks/developer-guide/security.md

```diff
@@ -51,6 +51,29 @@ For example:
           runAsUser: 1000
 ```
 
- The `runAsUser` field specifies that for any Containers in the Pod, all processes run with user ID 1000. The runAsGroup field specifies the primary group ID of 1000 for all processes within any containers of the Pod
+ The `runAsUser` field specifies that for any Containers in the Pod, all processes run with user ID 1000. The runAsGroup field specifies the primary group ID of 1000 for all processes within any containers of the Pod. If this field is not set, the primary group ID of the containers will be root. Using `runAsNonRoot: true` makes it impossible for containers to run as root, it will require a defined non-zero numeric USER directive defined in the container image.
```

Comment:
It forces the container to not be able to run with root inside the container.
Not "makes it impossible for containers to run as root"